### ISSUE TITLE
HTML Card Issues for photo-overlay

### ISSUE THREAD

**Issue by loretoparisi** [Scarygami/mirror-api#15]
For a card like this

```
    "{ "html": "<article class=\"photo\">\n  <img src=\"https://mirror-api-playground.appspot.com/links/filoli-spring-fling.jpg\" width=\"100%\" height=\"100%\">\n  <div class=\"photo-overlay\"/>\n  <section>\n    <p class=\"text-auto-size\">Spring Fling Fundraiser at Filoli</p>\n  </section>\n</article>\n", "notification": { "level": "DEFAULT" } }"
```

(taken from the Google Developers Glass Playground, https://developers.google.com/glass/playground

it seems that the photo-overlay is not being showed.
Is not implemented yet ?

A complex card like this

```
    "{ "html": "<article class=\"photo\">\n  <img src=\"https://mirror-api-playground.appspot.com/links/filoli-spring-fling.jpg\" width=\"100%\" height=\"100%\">\n  <div class=\"photo-overlay\"/>\n  <section>\n    <p class=\"text-auto-size\">Spring Fling Fundraiser at Filoli</p>\n  </section>\n</article>\n", "notification": { "level": "DEFAULT" } }"
```

works properly instead :+1: 

![schermata 2013-05-22 alle 23 39 58](https://f.cloud.github.com/assets/163333/551376/6b80fb66-c328-11e2-9951-6fc00f0f24d0.png)

![schermata 2013-05-22 alle 23 45 45](https://f.cloud.github.com/assets/163333/551395/08256ede-c329-11e2-8700-ea796c1b9016.png)

**Comment by Scarygami** [comment]
Works for me and looks exactly like the playground.

**Comment by loretoparisi** [comment]
Ok, let me try again I suppose is the bash escaping

``` javascript
curl --header "Content-Type:application/json" --header "Authorization: Bearer ya29.XXXXXXX" -d '{ "html": "<article class=\"photo\">\n  <img src=\"https://mirror-api-playground.appspot.com/links/filoli-spring-fling.jpg\" width=\"100%\" height=\"100%\">\n  <div class=\"photo-overlay\"/>\n  <section>\n    <p class=\"text-auto-size\">Spring Fling Fundraiser at Filoli</p>\n  </section>\n</article>\n", "notification": { "level": "DEFAULT" } }' "https://xxxxxx.appspot.com/_ah/api/mirror/v1/timeline"
{
 "updated": "2013-05-22T21:37:37.350350",
 "created": "2013-05-22T21:37:37.350370",
 "notification": {
  "level": "DEFAULT"
 },
 "html": "\u003carticle class=\"photo\"\u003e\n  \u003cimg src=\"https://mirror-api-playground.appspot.com/links/filoli-spring-fling.jpg\" width=\"100%\" height=\"100%\"\u003e\n  \u003cdiv class=\"photo-overlay\"/\u003e\n  \u003csection\u003e\n    \u003cp class=\"text-auto-size\"\u003eSpring Fling Fundraiser at Filoli\u003c/p\u003e\n  \u003c/section\u003e\n\u003c/article\u003e\n",
 "id": 33003,
 "isDeleted": false,
 "kind": "mirror#timelineItem",
 "etag": "\"GltmUmqwkaLBwPyK1uZRsoT0KU4/ahPDK6bGlKS7lu0ITkyApfE_XX4\""
}
```

**Comment by Scarygami** [comment]
I would suggest using the API Explorer for testing, since it makes things slightly easier. (unless you have specific reasons for using curl)
